gravity.sh

```diff
@@ -464,7 +464,7 @@ gravity_ParseFileIntoDomains() {
   # Determine if we are parsing a consolidated list
   #if [[ "${source}" == "${piholeDir}/${matterAndLight}" ]]; then
     # Remove comments and print only the domain name
-    # Most of the lists downloaded are already in hosts file format but the spacing/formating is not contigious
+    # Most of the lists downloaded are already in hosts file format but the spacing/formating is not contagious
```

Comment:
```suggestion
    # Most of the lists downloaded are already in hosts file format but the spacing/formating is not 
contiguous
```

![image](https://user-images.githubusercontent.com/1998970/74687981-ee841300-51cd-11ea-82b6-9beaee02f30c.png)